src/KafkaFlow.Retry.IntegrationTests/EmptyPartitionKeyRetryDurableTests.cs

```diff
@@ -0,0 +1,120 @@
+namespace KafkaFlow.Retry.IntegrationTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using KafkaFlow.Retry.IntegrationTests.Core.Bootstrappers.Fixtures;
+    using KafkaFlow.Retry.IntegrationTests.Core.Messages;
+    using KafkaFlow.Retry.IntegrationTests.Core.Producers;
+    using KafkaFlow.Retry.IntegrationTests.Core.Storages;
+    using KafkaFlow.Retry.IntegrationTests.Core.Storages.Assertion;
+    using KafkaFlow.Retry.IntegrationTests.Core.Storages.Repositories;
+    using Microsoft.Extensions.DependencyInjection;
+    using Xunit;
+
+    [Collection("BootstrapperHostCollection")]
+    public class EmptyPartitionKeyRetryDurableTests
```

Comment:
It's missing tests with the null partition key. But please check my other comment.

Reply:
we can't test with a null partition because the Kafka flow doesn't support it.

Reply:
You can test it if the test does not start with the production of a Kafka message but with a consumption. If we want to support null partition keys we should have a test for it.